fixtures/showtec/performer-2500.json

```
@@ -0,0 +1,225 @@
+      "name": "Fresnel",
+      "degreesMinMax": [10, 60]
+    }
+  },
+  "availableChannels": {
+    "Master Dimmer": {
+      "fineChannelAliases": ["Master Dimmer fine"],
+      "defaultValue": 0,
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Warm White": {
+      "fineChannelAliases": ["Warm White fine"],
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Warm White"
+      }
+    },
+    "Natural White": {
+      "fineChannelAliases": ["Natural White fine"],
+      "defaultValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "White"
+      }
+    },
+    "CCT": {
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 10],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [11, 35],
+          "type": "ColorTemperature",
+          "colorTemperatureStart": "warm",
+          "colorTemperatureEnd": "cold",
+          "comment": "2700 K"
+        },
+        {
+          "dmxRange": [36, 60],
+          "type": "ColorTemperature",
+          "colorTemperatureStart": "warm",
+          "colorTemperatureEnd": "cold",
+          "comment": "3000 K"
+        },
+        {
+          "dmxRange": [61, 85],
+          "type": "ColorTemperature",
+          "colorTemperatureStart": "warm",
+          "colorTemperatureEnd": "cold",
+          "comment": "3200 K"
+        },
+        {
+          "dmxRange": [86, 110],
+          "type": "ColorTemperature",
+          "colorTemperatureStart": "warm",
+          "colorTemperatureEnd": "cold",
+          "comment": "3400 K"
+        },
+        {
+          "dmxRange": [111, 135],
+          "type": "ColorTemperature",
+          "colorTemperatureStart": "warm",
+          "colorTemperatureEnd": "cold",
+          "comment": "3600 K"
+        },
+        {
+          "dmxRange": [136, 160],
+          "type": "ColorTemperature",
+          "colorTemperatureStart": "warm",
+          "colorTemperatureEnd": "cold",
+          "comment": "3800 K"
+        },
+        {
+          "dmxRange": [161, 185],
+          "type": "ColorTemperature",
+          "colorTemperatureStart": "warm",
+          "colorTemperatureEnd": "cold",
+          "comment": "4000 K"
+        },
+        {
+          "dmxRange": [186, 210],
+          "type": "ColorTemperature",
+          "colorTemperatureStart": "warm",
+          "colorTemperatureEnd": "cold",
+          "comment": "4200 K"
+        },
+        {
+          "dmxRange": [211, 235],
+          "type": "ColorTemperature",
+          "colorTemperatureStart": "warm",
+          "colorTemperatureEnd": "cold",
+          "comment": "4500 K"
+        },
+        {
+          "dmxRange": [236, 255],
+          "type": "NoFunction"
+        }
+      ]
+    },
+    "Strobe": {
+      "defaultValue": 0,
+      "capabilities": [
+        {
+          "dmxRange": [0, 9],
+          "type": "NoFunction"
```

"No Function" in strobe channels should be explicit, since it could mean both "lamp constantly on" or "lamp constantly off".

```suggestion
          "type": "ShutterStrobe",
          "shutterEffect": "Open"
```

---

Good point. From reading the manual, §6.6.2 point 3 says:
> the adjustment range is between 0-25, from OFF to high frequency

This suggests to me that 'No Fuction' would mean the strobe function is off, meaning the lamp is contantly on, however it's not clear.

---

If there is no explicit mention, it's probably always on, but still it's better to be explicit. I'd say let's change it all capabilities of this channel.

(Also, please update the dates to today before merging.)